app/views/flights/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<% @flights.each do |flight| %>
+  <section class="flight-<%=flight.id%>">
```

Comment:
Should be an id, not a class. Classes in css are more general whereas the ids are unique. Will work the same here, but just a note.:)